src/SUMMARY.md

```diff
@@ -1,6 +1,8 @@
 # Rust Forge
 
 [Overview](./README.md)
+- [Chat Platforms](./chat/README.md)
+  - [Zulip](./chat/zulip.md)
```

Comment:
We should probably add at least a "WIP" page for Discord if we add this section =)